src/storage/mutate/UpdateVertexProcessor.cpp

```diff
@@ -95,6 +96,11 @@ kvstore::ResultCode UpdateVertexProcessor::collectVertexProps(
         return ret;
     }
 
+    auto status = this->schemaMan_->toTagName(this->spaceId_, tagId);
+    if (!status.ok()) {
+        LOG(ERROR) << "toTagName failed, tagId: " << tagId;
+        return kvstore::ResultCode::ERR_UNKNOWN;
+    }
```

Comment:
I think graphed check the validity of tag. storage should not do it.

Reply:
You can't make sure the schema is the same as graph